CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [8.8.x] - 2025-01-??
 
 - Added: Experimental preset option to place all majors or pickups logically. This makes sure pickups that are not required can be collected.
+- Fixed: The map tracker no longer opens if opening it was cancelled.
```

Comment:
Currently sounds a bit weird. Maybe this instead?
```suggestion
- Fixed: The map tracker no longer opens if the save station prompt was cancelled.
```

Reply:
I'd use starting location instead of save station here. Not every game has save stations

Reply:
Right.